packages/automated-dispute/src/services/eboProcessor.ts

```diff
@@ -48,6 +62,42 @@ export class EboProcessor {
         }, msBetweenChecks);
     }
 
+    /**
+     * Check if all the modules have been granted approval within the accounting module.
+     *
+     * @throws {PendingModulesApproval} when there is at least one module pending approval
+     */
+    private async checkAllModulesApproved() {
+        const approvedModules: Address[] =
+            await this.protocolProvider.getAccountingApprovedModules();
+
+        const summary: Record<"approved" | "notApproved", Partial<AccountingModules>> = {
+            approved: {},
+            notApproved: {},
+        };
+
+        for (const [moduleName, moduleAddress] of Object.entries(this.accountingModules)) {
```

Comment:
```suggestion
        for (const key in this.accountingModules) {
```
You can do it this way also, just to let you know, not requesting any changes here 🤣